### PR TITLE
Assignment more crud

### PR DESCRIPTION
**Pull request by benjaminmodayil**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,6 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+# Ignore Figaro's application.yml file
+/config/application.yml
```
Comment (alexstophel) [inline .gitignore]: We should add this so your API keys aren't available online.

**Comment by alexstophel** [comment]
I removed application.yml and squashed my commit into yours to remove the file from the history on Github. We don't want anyone else to see your API keys for security reasons. I'm going to go ahead and merge. Be sure to ```git pull origin master``` on the ```master``` branch before starting the next checkpoint

**Comment by benjaminmodayil** [comment]
Okay, I believe I got it all working. Something went wrong initially when I
was going in between two computers, but it all seems to be working again.
Thanks.

**Comment by alexstophel** [comment]
Cool!

**Pull request merged**